controller.go

```diff
@@ -154,6 +155,7 @@ type controller struct {
 	agentInitDone          chan struct{}
 	keys                   []*types.EncryptionKey
 	clusterConfigAvailable bool
+	plugingetter           getter.PluginGetter
```

Comment:
I think we no longer need this member in `controller`, rather we can just keep it confined to drvRegistry